ambuda/logger.py

```diff
@@ -0,0 +1,21 @@
+"""Providers logger utilities."""
+import logging
+
+from config import BaseConfig
+
+
+class Logger(logging.Logger):
+    LOG_LEVEL = 0
```

Comment:
The default log level is `0` which is `logging.NOTSET` (https://github.com/python/cpython/blob/main/Lib/logging/__init__.py#L105). Shall we instead pick a reasonable default, such as `logging.INFO`?